src/functionalTest/resources/scenarios/RIA-1152-&-1153-generate-hearing-ready-bundle.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Think this filename should not include `generate-hearing-ready-bundle`

Reply:
sorted